.buildkite/pipeline.yml

```diff
@@ -759,6 +759,15 @@
     - rm -rf ./python/ray/thirdparty_files; rm -rf ./python/ray/pickle5_files; ./ci/travis/ci.sh build
     - bazel test --config=ci $(./scripts/bazel_export_options) --build_tests_only python/ray/workflow/... python/ray/data/...
 
+- label: ":potable_water: Workflow & Dataset tests (Pandas) (Python 3.7)"
```

Comment:
Can we actually enable the flag by default in this PR for from_pandas() at least? We shouldn't generally make separate CI builders except for major features.

In a followup PR, we can enable pandas-type blocks in more scenarios as an option / config.

Reply:
Done.